src/Utils/TemplateEngine.php

```diff
@@ -116,9 +116,9 @@ public function compile( $target, array $args = [] ) {
 
 		$complied = $this->container[$target];
 		
-		foreach ($args as $key => $value) {
-			if ($value !== null) {
-				$complied = str_replace(['{{' . $key . '}}', '{{#' . $key . '}}'], $value, $complied);
+		foreach ( $args as $key => $value) {
+			if ( $value !== null) {
+				$complied = str_replace( ['{{' . $key . '}}', '{{#' . $key . '}}'], $value, $complied);
```

Comment:
@StarsRivers the spaces must be present in both sides of the code, no only in the left side